module2/homework/transformContainers/transform.cpp

```diff
@@ -0,0 +1,29 @@
+#include <algorithm>
+#include <iostream>
+
+#include "tranform.hpp"
+
+std::map<int, std::string> removeDuplicateAndTranformToMap(
+    std::list<std::string>& uniqueList,
+    std::deque<int>& uniqueDeque) {
```

Comment:
I know I adviced you to use reference here, but I re-think this advice and it was wrong idea.
In current solution, the original input conteiners,  list and deque are modified, during e.g. sorting. I think this function should not modify original vakues, so passing by copy is better here.
Sorry for that.